docs/src/main/asciidoc/native-reference.adoc

```diff
@@ -2133,23 +2133,50 @@ $ ./mvnw package -DskipTests -Dnative \
 https://docs.oracle.com/javacomponents/jmc-5-4/jfr-runtime-guide/about.htm#JFRUH170[Java Flight Recorder (JFR)] and
 https://www.oracle.com/java/technologies/jdk-mission-control.html[JDK Mission Control (JMC)]
 can be used to profile native binaries since GraalVM CE 21.2.0.
-However, JFR in GraalVM is currently significantly limited in capabilities compared to HotSpot.
-The custom event API is fully supported, but many VM level features are unavailable.
-They will be added in future releases. Current limitations are:
-
-* Minimal VM level events
+However, JFR in GraalVM is currently limited in capabilities compared to HotSpot.
+The custom event API is fully supported, but some VM level features are unavailable.
+More events and JFR features will continue to be added in later releases.
+The following table outlines Native Image JFR support and limitations by version.
+
+[cols="2,5a,5a"]
+|===
+|GraalVM Version |Supports |Limitations
+
+|GraalVM CE 21.3
```

Comment:
@roberttoyonaga can you please add Mandrel alongside like you did in the building-native-image.doc?